### DISCUSSION TITLE
Process Modules

### DISCUSSION THREAD

**Discussion by gabriel-maxx** [winsiderss/systeminformer#1102] [Questions / Answers] [answered]
hello I would like to know how I do it if I want to load a module automatically and permanently in a process? I meant when I load a dll in a process is it temporary after restarting the process or does the computer have to do it all over again... is there some sort of configuration somewhere like registry that manages these things?

**Comment by dmex** [comment, marked as answer]
> when I load a dll in a process is it temporary

Yes it's temporary until the process exits or machine reboots.
